zest/releaser/__init__.py

```diff
@@ -1,4 +1,10 @@
 import pkg_resources
+from colorama import init
 
+# Initialize colorized output.  Set it to reset after each print, so
+# for example a foreground color does not linger into the next print.
+# Note that the colorama docs say you should call `deinit` at exit,
+# but it looks like it already does that itself.
+init(autoreset=True)
```

Comment:
Should this `init()` go here? This way we have another package's init stuff Real High Up in our code.

For explicitness' sake I'd prefer to have it in the same place where we set up logging (probably the `main()` in pre/postrelease.py)